msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
```

Comment:
Suggestion for reworking this section to incorporate latest comments:

#### Mobile debugging

Here's how to remotely debug your app running in Office app for Android. 

1. From your Android device, launch the Office app.
2. Open your profile **Me** > **Settings** > **Allow debugging**, and toggle on the option for **Enable remote debugging**.

[screenshot showing Enable remote debugging]

3. Exit **Settings**, then exit your profile screen, **Me**.
4. Select **Apps** and then launch your sideloaded app so that its running within the Office app.
6. From your dev machine, open your browser to its DevTools inspection page. For example, go to `edge://inspect/#devices` in Microsoft Edge to display a list of debug-enabled Android WebViews.
7. Find the `Microsoft Teams Tab` with your tab URL and select **inspect** to start debugging your app with DevTools.

[screenshot showing list of webviews in devtools]

8. Debug your tab app within the Android WebView the same way you would [remotely debug](/microsoft-edge/devtools-guide-chromium/remote-debugging) a regular website on an Android device.

Reply:
@erikadoyle step#5 should have prerequisite that mobile phone should be connected to dev machine and usb debugging is allowed.  

https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/remote-debugging

- Open the Developer Options screen on your Android. See [Configure On-Device Developer Options](https://developer.android.com/studio/debug/dev-options).
- Select Enable USB Debugging.